x-pack/plugins/lens/public/app_plugin/lens_top_nav.tsx

```diff
@@ -71,6 +73,23 @@ function getLensTopNavConfig(options: {
         defaultMessage: 'Save',
       });
 
+  if (contextOriginatingApp) {
+    topNavMenu.push({
+      label: i18n.translate('xpack.lens.app.goBackLabel', {
+        defaultMessage: `Go back to {contextOriginatingApp}`,
+        values: { contextOriginatingApp },
```

Comment:
If I followed the code correctly: The originating app is not i18n-ified (it will be the actual app id). Can we somehow pass in an i18n-ified string?